src/scenic/syntax/scenic.gram

```diff
@@ -1206,7 +1206,8 @@ conjunction (memo):
     | inversion
 
 inversion (memo):
-    | 'not' a=inversion { ast.UnaryOp(op=ast.Not(), operand=a, LOCATIONS) }
+    # [SCENIC NOTE]: Fail `not visible` to be handled later
+    | 'not' !"visible" a=inversion { ast.UnaryOp(op=ast.Not(), operand=a, LOCATIONS) }
```

Comment:
This makes `visible` a hard keyword in practice, since you couldn't use it as a boolean variable and then write `x = not visible`. To get better error messages we should probably just make it an actual hard keyword if we go this route; alternatively, we could consider changing the syntax `not visible` to `invisible` (but I think it could be confusing this way).